ansible/provision-node.yml

```diff
@@ -0,0 +1,15 @@
+# This playbook provisions a Redpanda cluster with no bells or whistles for evaluation purposes
+# variables
+# advertise_public_ips : causes the cluster to set a public ip on advertised_kafka_api, allowing access to the cluster from the public internet
+---
+- name: provision nodes
+  hosts: redpanda
+  vars:
+    advertise_public_ips: true
+    prep_data_dir: true
+    prep_deps: true
```

Comment:
I think we should be setting the default to true for the node prep stuff in it's own tasks rather than doing it this way. I can't think of a reason where we would really ever turn this off.

Reply:
It isn't included by default in the original role, so I didn't want to include it by default in the new role. I also think there is a clear scenario where someone would want these off: golden image with all necessary RP deps and R2G data dir. If someone had that and we started mucking about, they'd be annoyed.

Reply:
I think this is one area where we need to be opinionated for supportability reasons. We don't want people creating mounts that aren't xfs, for example.

Reply:
I think if we were starting greenfield I'd agree, but we aren't. So it doesn't make sense to have functionality enabled that wasn't enabled in the past.

Reply:
resolved in this commit https://github.com/redpanda-data/deployment-automation/pull/156/commits/a0e885c35a43be527c844671a05dde3a9bafcbf8